addons/l10n_syscohada/i18n/l10n_syscohada.pot

```diff
@@ -0,0 +1,7802 @@
+#: model:account.account.template,name:l10n_syscohada.pcg_9131
+msgid "Counterparties - Forward Commodity Purchases"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9171
+#: model:account.account,name:l10n_syscohada.2_pcg_9171
+#: model:account.account.template,name:l10n_syscohada.pcg_9171
+msgid "Counterparties - Forward Commodity Sales"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9132
+#: model:account.account,name:l10n_syscohada.2_pcg_9132
+#: model:account.account.template,name:l10n_syscohada.pcg_9132
+msgid "Counterparties - Forward currency purchases"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9172
+#: model:account.account,name:l10n_syscohada.2_pcg_9172
+#: model:account.account.template,name:l10n_syscohada.pcg_9172
+msgid "Counterparties - Forward currency sales"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9114
+#: model:account.account,name:l10n_syscohada.2_pcg_9114
+#: model:account.account.template,name:l10n_syscohada.pcg_9114
+msgid "Counterparties - Issuance facilities"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9124
+#: model:account.account,name:l10n_syscohada.2_pcg_9124
+#: model:account.account.template,name:l10n_syscohada.pcg_9124
+msgid "Counterparties - Items endorsed by third parties"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9151
+#: model:account.account,name:l10n_syscohada.2_pcg_9151
+#: model:account.account.template,name:l10n_syscohada.pcg_9151
+msgid "Counterparties - Loans granted but not disbursed"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9112
+#: model:account.account,name:l10n_syscohada.2_pcg_9112
+#: model:account.account.template,name:l10n_syscohada.pcg_9112
+msgid "Counterparties - Loans still to be collected"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9158
+#: model:account.account,name:l10n_syscohada.2_pcg_9158
+#: model:account.account.template,name:l10n_syscohada.pcg_9158
+msgid "Counterparties - Other financing commitments granted"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9118
+#: model:account.account,name:l10n_syscohada.2_pcg_9118
+#: model:account.account.template,name:l10n_syscohada.pcg_9118
+msgid "Counterparties - Other financing commitments obtained"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9168
+#: model:account.account,name:l10n_syscohada.2_pcg_9168
+#: model:account.account.template,name:l10n_syscohada.pcg_9168
+msgid "Counterparties - Other guarantees granted"
+msgstr ""
+
+#. module: l10n_syscohada
+#: model:account.account,name:l10n_syscohada.1_pcg_9128
+#: model:account.account,name:l10n_syscohada.2_pcg_9128
```

Comment:
we might try to avoid those account.account,name by using a db where another loczalition is installed and then we install l10n_syscohada so the coa is not instantiated

Reply:
And so what should i put instead ?

Reply:
@jco-odoo ?

Reply:
nothing, we should just have L70: #: model:account.account.template,name:l10n_syscohada.pcg_164

Reply:
@jco-odoo Ok then the PR is ready i think :+1: